conan/cli/args.py

```diff
@@ -15,6 +15,8 @@
                        pattern uses 'fnmatch' style wildcards.
     --build=![pattern] Excluded packages, which will not be built from the source, whose package
                        reference matches the pattern. The pattern uses 'fnmatch' style wildcards.
+    --build=~[pattern] Excluded packages, which will not be built from the source, whose package
```

Comment:
If we are going to converge to using ``~``, maybe remove the ``!`` from the docstring?